zapisy/apps/enrollment/courses/templates/courses/course_parts/course_head.html

```diff
@@ -15,6 +15,17 @@ <h1 class="d-inline-block">
     </div>
 </div>
 
+<style>
+    #table-info td {
+        width: 100%; 
+        word-wrap: break-word; 
+    }
+
+    #table-info th {
+        white-space: nowrap;
+    }
```

Comment:
Teraz jest faktycznie dosyć sztywno, ale takie coś chyba zarówno robi, co trzeba, jak i jest bardziej elastyczne. Nie wiem, czy nie wiążą się z tym jakieś kruczki, czy nie warto dodać czegoś jeszcze...
```suggestion
    #table-info th {
        min-width: max-content; 
    }
```